stubs/config.php

```diff
@@ -70,4 +70,12 @@
         //  ],
     ],
 
+    'requirements' => [
```

Comment:
Also right now we have our mins defined in our `composer.json`, could you change that, so we use this new feature?